Makefile

```
@@ -1,7 +1,7 @@
 # SeedSync Makefile - Docker Only
 # Simplified build system for containerized deployment
 
-.PHONY: build run stop logs clean test
+.PHONY: build run stop logs clean test test-image
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)